triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBag.java

```diff
@@ -154,9 +154,9 @@ private String tabulate(String title, BiFunction<PropertyBag, String, String> ge
             String format = String.format("%%-%ds", lenPad);
 
             // Performance : avoid using concat() -> use a StringBuilder instead.
-            return new StringBuilder(title).append(lineSeparator).append(propertyNames.stream()
-                    .map(n -> new StringBuilder(INDENTATION).append(String.format(format, n)).append(" : ").append(getValue.apply(this, n)).toString())
-                    .collect(Collectors.joining(lineSeparator))).toString();
+            return title + lineSeparator + propertyNames.stream()
+                .map(n -> INDENTATION + String.format(format, n) + " : " + getValue.apply(this, n))
+                .collect(Collectors.joining(lineSeparator));
```

Comment:
This change should be reverted.
If you look at the comment, using the `StringBuilder` instead of concats (or `+`) was done for performance purposes.